packages/-ember-data/tests/dummy/app/routes/application/route.ts

```diff
@@ -1,16 +1,21 @@
 import Route from '@ember/routing/route';
+import { inject } from '@ember/service';
+
+type Store = import('@ember-data/store').default;
```

Comment:
Any objection to switching to `import type` for new stuff like this? Less work to do later.

```suggestion
import type Store from '@ember-data/store';
```

Reply:
I've opened an issue for it and someone has agreed to take it on. I think it's as simple as removing our plugin and configuring the new eslint-typescript rules which autofix, then running the fix. I'm going to let them have a chance to have a stab at it before I jump in though.